app/models/user.rb

```diff
@@ -45,4 +45,8 @@ def remember_me
   def signed_hiring_agreement?
     HiringAgreements::Term.signed_by?(self)
   end
+
+  def permissions
+    Businesses::Permission.new(subscriptions)
```

Comment:
We use this in a bunch of other places in the app. If we are going to make the refactor we should also update the rest of the codebase.

Let me know if you'd like to do that in this PR or a future one. If we do it in a future one then let's remove this abstraction from this PR and add it all in at once.

Reply:
I think that it's better to introduce this abstraction in a different PR.